pydisdrometer/DSDProcessor.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, wl=tmatrix_aux.wl_X, dr =1, shape='bc'):
         self.scatterer = Scatterer(wavelength=wl, m=refractive.m_w_10C[wl])
         self.scatterer.psd_integrator = PSDIntegrator()
         self.scatterer.psd_integrator.axis_ratio_func = lambda D: 1.0/DSR_list[shape](D)
-        self.scatterer.psd_integrator.D_max = 10.0
+        self.scatterer.psd_integrator.D_max = 9.0
         self.scatterer.psd_integrator.geometries = (tmatrix_aux.geom_horiz_back,
```

Comment:
Yes, this was to avoid a bug in the fortran code for large drop sizes at W-Band. 10 makes this bug appear less frequently.